### PR TITLE
Release 1.15.0

### PR DESCRIPTION
**Pull request by tvaron3**
Release 1.15.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ## Release History
+### 1.15.0 (2024-04-17)
```
Comment (kushagraThapar) [inline CHANGELOG.md:2]: if we are going to release today, we should update the date for today.

**Review by kushagraThapar — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (tvaron3) [inline CHANGELOG.md:2]:
```suggestion
### 1.15.0 (2024-04-18)
```

**Review by kushagraThapar — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (tvaron3) [inline CHANGELOG.md:2]:
```suggestion
### 1.15.0 (2024-04-18)
```

Comment (tvaron3) [reply CHANGELOG.md]: changed

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

**Review by kushagraThapar — APPROVED** [review]

**Pull request merged**